### DISCUSSION TITLE
Is Molstar available to use in production application?

### DISCUSSION THREAD

**Discussion by irfanfs** [molstar/molstar#140] [General]
Is Molstar available to use in production application? I am currently working on rendering the 3d molecular structures so looking out for some good npm package to achieve this.

**Comment by dsehnal** [comment]
Yes. It's been used in production at RCSB (e.g. https://www.rcsb.org/3d-view/1TQN/1), PDBe, and other places for quite some time now. The NPM package is https://www.npmjs.com/package/molstar
